src/content/onlineservice/atrailers_content_handler.cc

```diff
@@ -129,35 +129,27 @@ std::shared_ptr<CdsObject> ATrailersContentHandler::getObject(const pugi::xml_no
 
     item->setClass("object.item.videoItem");
 
-    temp = info.child("rating").text().as_string();
-    if (!temp.empty())
-        item->setMetadata(M_RATING, temp);
-
-    temp = info.child("studio").text().as_string();
-    if (!temp.empty())
-        item->setMetadata(M_PRODUCER, temp);
-
-    temp = info.child("director").text().as_string();
-    if (!temp.empty())
-        item->setMetadata(M_DIRECTOR, temp);
+    auto propertyMap = std::map<std::string, metadata_fields_t> {
```

Comment:
This looks like it can be replaced with std::vector<std::pair<>>. structured bindings still work.